src/core/apparmor-setup.c

```diff
@@ -0,0 +1,98 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+#include <unistd.h>
+
+#include "fd-util.h"
+#include "fileio.h"
+#include "log.h"
+#include "macro.h"
+#include "apparmor-setup.h"
+#include "apparmor-util.h"
+#include "string-util.h"
+
+#if HAVE_APPARMOR
+#include <sys/apparmor.h>
+#endif
+
+#define APPARMOR_PROFILE_NAME "systemd"
+#define APPARMOR_PROFILE_ETC_PATH "/etc/systemd/profile.apparmor"
+#define APPARMOR_PROFILE_USR_PATH "/usr/lib/systemd/profile.apparmor"
```

Comment:
quite frankly I don't see the point of these defines. The latter two are used exactly once, and it would be more readable to just specify them literally where they are used. And the former is simple enough to not result in typos if repeated, hence I don't think it warrants a macro here..

I mean, macros are great to abbreviate stuff, to lower the risk of making typos, to make things configurable. But neither of those reasons apply here, hence please don't add the extra indirection that these macros are.